templates/index.html

```diff
@@ -72,4 +73,4 @@ <h2>Build on Matrix</h2>
         </div>
     </div>
 </div>
-{% endblock content %}
+{% endblock content %}
```

Comment:
the EOF newline disappeared 🙀

Reply:
:o

Reply:
Great reminder to reinstall the editorconfig plugin /o\\